src/functions/Pester.SessionState.Mock.ps1

```diff
@@ -113,6 +112,14 @@ is to be mocked.  This should be a module that _calls_ the mocked
 command; it doesn't necessarily have to be the same module which
 originally implemented the command.
 
+.PARAMETER RemoveParameterType
+Optional list of parameter names in the original command
+that should not keep the same type-requirement in the mock.
+
+.PARAMETER RemoveParameterValidation
+Optional list of parameter names in the original command
+that should not have the same valdiation rules in the mock.
+
```

Comment:
```suggestion
.PARAMETER RemoveParameterType
Optional list of parameter names that should use Object as the parameter
type instead of the parameter type defined by the function. This relaxes the
type requirements and allows some strongly typed functions to be mocked
more easily.

.PARAMETER RemoveParameterValidation
Optional list of parameter names in the original command
that should not have any validation rules applied. This relaxes the 
validation requirements, and allows functions that are strict about their
parameter validation to be mocked more easily.

```